### ISSUE TITLE
Library size is wildly inaccurate

### ISSUE THREAD

**Issue by BRTPOB** [CyferShepard/Jellystat#184]
**Describe the bug**
When viewing the Libraries tab, and looking at the different libraries the size of my libraries is wildly inaccurate, and all of them are undersized rather than oversized.

**Environment Details (please complete the following information):**
 - OS: Ubuntu Docker
 - Browser: Edge
 - Jellystat Version: 1.0.9
 - Jellyfin Version: 10.8.13

**To Reproduce**
1. Browse to Jellystat instance.
2. Login
3. Click on Libraries
4. View Library Size

**Expected behavior**
Expected library size to actually show the full size of each library.

**Screenshots**
![image](https://github.com/CyferShepard/Jellystat/assets/7201028/c743ec88-dc55-4986-bdc5-6fd8a12b41db)

**Task Logs**
If applicable, add Task Logs for your problem.

**Container Logs**
If applicable, add Container Logs for your problem.

**Additional context**
The actual library size that I have is 17TB, so there's quite a bit missing.

**Comment by CyferShepard** [comment]
Hi @BRTPOB , can you try the `unstable` tag? a fix was pushed there to address the issue. let me know if it helps. This will be making its way to stable within the next week or so

**Comment by CyferShepard** [comment]
Make sure to do a resync on unstable and then check the updated stats

**Comment by CyferShepard** [comment]
maybe try restarting the container, also check the docker logs for any issues

**Comment by BRTPOB** [comment]
I've restarted the container a few times, including after updates. I just restarted it again, immediately ran the docker logs to verify it as I forced a sync and it's still completely out of date.

My DV TV folder is 13TB, but Jellystat is only showing 1.21. I also noted that when I attempted to run the full sync, it gets to step 4/6, so hitting my regular TV Shows library which is the largest, runs for a short period and then just stops. I can't see how to enable debug logging to verify the different steps occurring, but it seems like it's not actually reading in the full size of the libraries still, and it's failing to parse something that stops it from completing the process.

**Comment by CyferShepard** [comment]
are there any errors in the docker logs, the task logs or in the web console?

**Comment by BRTPOB** [comment]
No errors in the docker logs, and the web console just says 0 second (which is a separate bug) plus "Running". If I go look back, yesterday it said it failed but the time it took was also 0 second.

**Comment by BRTPOB** [comment]
No worries. I've got more than enough RAM, as it's running with 128GB currently, but I'm not sure how much the app is limited to.

**Comment by CyferShepard** [comment]
So its a memory limitation within the backend framework, which limits it by default to 1.5gigs of ram, and with general overhead you only end up with maybe a gig of usable memory. I could always increase the default but id rather make it work a bit smarter so it doesn't need as much memory. Because the more items it tries to process at the same time the more memory itl need.

**Comment by BRTPOB** [comment]
Yeah, that makes sense, I more meant "if it's trying to access more than X it shouldn't be a problen on my end, because I've got a whole lot of headroom."

**Comment by CyferShepard** [comment]
So this one seems like it the connection to jellyfin got reset. Although it gives a generic reason, could be anything including a jellyfin restart or a connection timeout

**Comment by BRTPOB** [comment]
Which is odd, because both containers are running on the same system, and there were no reboots of either container that I saw. Is there a way of doing debug logging on Jellystat to try and pinpoint exactly what's going on?

**Comment by CyferShepard** [comment]
not really, this stack trace is as close as you can get, maybe theres some option in axios (the library used for api calls), will look into it

**Comment by BRTPOB** [comment]
I just wanted to update you and let you know that some of my libraries have now updated, and they seem to be showing a pretty accurate count though others aren't. I'm guessing it's still from the timeout.

**Comment by dinosmm** [comment]
I am seeing the same (?) issue using Jellystat 1.1.5 with Emby.
The item counts are correct, but the size is much lower than reality. For example, Jellystat says my Movies library is 6.45TB, but it's much higher than that (nearly triple). 
Could this be related to movies with multiple versions? Could Jellystat only be counting the 1080p version (as I have noticed a return of an old bug which shows the 1080p version even when playing the 4K version)?
